app/javascript/app/components/country/country-ghg-emissions/country-ghg-emissions-selectors.js

```diff
@@ -117,15 +117,18 @@ export const getVersionSelected = createSelector(
   [getVersionOptions, getVersionSelection],
   (versions, selected) => {
     if (!versions || !versions.length) return {};
```

Comment:
I think returning a null here will improve the cache system of reselect and also avoid to check the `isEmpty` function in next selectors